src/core/Akka.Tests.Performance/Actor/AddressSpec.cs

```diff
@@ -25,6 +25,7 @@ public void Setup(BenchmarkContext context)
         public void ParseLocalThroughput(BenchmarkContext context)
         {
             Address.Parse("akka.tcp://sys@localhost:9091");
+            _parseThroughput.Increment();
```

Comment:
Calls the NBench `Counter` object that's used to measure the throughput of the `Address.Parse` function, which I omitted by accident before.